views/administration/school.hbs

```diff
@@ -380,6 +380,7 @@
 										type="checkbox"
 										name="rocketChat"
 										value="true"
+                                        data-testid="school-admin-activate-chat-checkbox"
```

Comment:
as we are already in the context of administration and its not only for activating it I would either name it `chat-checkbox` or if you would like to keep admin stuff, then `school-administration-chat-checkbox` or `administration-chat-checkbox`

Reply:
Thanks for the feedback.
So I updated the name to -->  school-administration-chat-checkbox